lettuce/django/management/commands/harvest.py

```diff
@@ -35,110 +34,103 @@
 from lettuce.django.server import LettuceServerException
 
 
+DJANGO_VERSION = StrictVersion(django.get_version())
+
+
 class Command(BaseCommand):
     help = u'Run lettuce tests all along installed apps'
     args = '[PATH to feature file or folder]'
-    requires_model_validation = requires_system_checks = False
```

Comment:
this line gives an error on Django==1.8, so that's why I replaced it below with something smarter